File: nhglib-core/src/io/github/voidzombie/nhglib/graphics/scenes/Scene.java
package io.github.voidzombie.nhglib.graphics.scenes;

import com.badlogic.gdx.utils.Array;
import io.github.voidzombie.nhglib.Nhg;
import io.github.voidzombie.nhglib.assets.Asset;

/**
 * Created by Fausto Napoli on 15/12/2016.
 */
public class Scene {
    public String name;
    public SceneGraph sceneGraph;

    public Array<Asset> assets;

    public Scene(Nhg nhg, String rootId) {
        sceneGraph = new SceneGraph(nhg, rootId);
        assets = new Array<>();
    }
}

File: nhglib-core/src/io/github/movementspeed/nhglib/graphics/scenes/Scene.kt
package io.github.movementspeed.nhglib.graphics.scenes

import com.badlogic.gdx.utils.Array
import io.github.movementspeed.nhglib.Nhg
import io.github.movementspeed.nhglib.assets.Asset

/**
 * Created by Fausto Napoli on 15/12/2016.
 */
class Scene(nhg: Nhg, rootId: String) {
    var name: String? = null
    var sceneGraph: SceneGraph

    var assets: Array<Asset>

    init {
        sceneGraph = SceneGraph(nhg, rootId)
        assets = Array()
    }
}
